docs/src/pages/demos/autocomplete/IntegrationAutosuggest.js

```diff
@@ -74,17 +74,17 @@ function renderSuggestion(suggestion, { query, isHighlighted }) {
   return (
     <MenuItem selected={isHighlighted} component="div">
       <div>
-        {parts.map((part, index) => {
-          return part.highlight ? (
```

Comment:
This is not just "removing unnecessary rules". This clearly adds a new rule. Could you open a different PR so we can discuss this?

Reply:
I did make the change. My strategy was the following:
- Step 1 remove all the rules from the .eslintrc configuration. 💯  rely on the default values of Airbnb.
- Step 2 add the rules back one by one in order to get a green build. Eventually, I have made some changes to the code. Like the array pattern or the test description duplication.

Reply:
I believe that the less code we have in the `.eslintrc` file, the better. I think that it's more important to follow a community default than micro-optimizing each rule.

Reply:
@oliviertassinari I can see the changes are quite reasonable. There's less code and it's more concise

Reply:
You should rebase your pull-request. This was already merged on master.

Reply:
@oliviertassinari Sorry but how do I do so? Like creating a whole new branch?